library/README.md

```diff
@@ -23,8 +23,8 @@ ace-admin theme                   | 1.4.0     | customized
 moment.js                         | 2.15.2    | unpatched
 bootstrap-datetimepicker          | 4.17.47   | unpatched
 dropzone.js                       | 5.5.0     | unpatched
-chart.js                          | 2.7.3     | unpatched
-chartjs-plugin-colorschemes       | 0.3.0     | unpatched
+chart.js                          | 2.8.0     | unpatched
```

Comment:
Given that the library is scoped to the plugin, should we have it listed there?  I can see benefits to both models.

Reply:
I thought about it, but considering that MantisGraph is an official, bundled plugin, I left it there. I'm fine with removing it from this list if you prefer.

Reply:
@vboctor let me know if I you want me to remove the entries from the library/README.md file. Without reply from you, I'll leave them in.